assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -1,10 +1,8 @@
 <div
-  row class="log-timestamp"
-  ng-if="(!chromeless) && state=='logs' && timeStart">
-  <div>
-    <span>Current log from {{timeStart}}&nbsp;</span>
-    <span ng-if="end">to {{timeEnd}}</span>
-  </div>
+  row cross-axis="end"
+  class="log-timestamp"
+  ng-if="(!chromeless) && state=='logs'">
+  <div ng-transclude><!-- expect start time, end time, status details --></div>
```

Comment:
The transclude is preventing the status from showing until the log starts streaming.